iron-autogrow-textarea.html

```diff
@@ -78,7 +78,6 @@
         autocomplete$="[[autocomplete]]"
         autofocus$="[[autofocus]]"
         inputmode$="[[inputmode]]"
-        name$="[[name]]"
```

Comment:
It would not work in an `iron-form` in any case, because it's not registering itself. You're right that it wouldn't work in a native form anymore. It still has a `name` property, so I could just add the form behaviour to it, which would make it work in an `iron-form`, and remove it from `paper-textarea` (since that just passes it down), to avoid duplication.